app/src/main/res/layout/fragment_prizes.xml

```diff
@@ -229,6 +230,43 @@
                 app:layout_constraintStart_toStartOf="@+id/textView35"
                 app:layout_constraintTop_toTopOf="@+id/textView9" />
 
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:alpha="0.8"
+                android:text="@string/Rs2000"
+                android:textColor="@android:color/white"
+                android:textSize="20sp"
+                app:layout_constraintBottom_toBottomOf="@+id/textView10"
+                app:layout_constraintEnd_toEndOf="@+id/textView23"
+                app:layout_constraintStart_toStartOf="@+id/textView23"
+                app:layout_constraintTop_toTopOf="@+id/textView10" />
+
+            <TextView
+                android:id="@+id/textView10"
```

Comment:
Another minor recommendation, although it's okay to use the autogenerated ids when in constraint layout, for the sake of consistency, use snake case for ids.